wordpress-mobile/WordPress-iOS
# Issue/6826 remove app review request for internal builds

**Fixes #6826** 
App review prompts will not appear for internal builds.

**To test:**
You can create an internal build and install it or use `_overridePromptingDisabledLocal` in  `AppRatingUtility`

1. Delete the previously installed version
2. Go to the Notifications screen and navigate into 5 notifications
  a. For internal builds or when the settings is overwritten you should **NOT** see the prompt to give a review.
  b. For any other case you should.

Needs review: @koke
